<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorElevationPropertiesWidgetBase</class>
 <widget class="QWidget" name="QgsVectorElevationPropertiesWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Elevation Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="mElevationGroupBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title">
      <string>Elevation Clamping</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">vectorgeneral</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,0">
      <item row="2" column="0">
       <widget class="QLabel" name="mLabelScale">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="mLabelClampingExplanation">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mOffsetLabel">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QgsDoubleSpinBox" name="mOffsetZSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QgsDoubleSpinBox" name="mScaleZSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QgsPropertyOverrideButton" name="mOffsetDDBtn">
        <property name="text">
         <string>…</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QComboBox" name="mComboClamping"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mBindingGroupBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title">
      <string>Elevation Binding</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">vectorgeneral</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="mComboBinding"/>
      </item>
      <item row="2" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="mLabelBindingExplanation">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mExtrusionGroupBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title">
      <string>Enable Extrusion</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">vectorgeneral</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsDoubleSpinBox" name="mExtrusionSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QgsPropertyOverrideButton" name="mExtrusionDDBtn">
        <property name="text">
         <string>…</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="mLabelExtrusionExplanation">
        <property name="text">
         <string>Extrusion controls how high features extend vertically above their base.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Profile Chart Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
      <item row="0" column="1">
       <widget class="QComboBox" name="mTypeComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Interpretation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="mInterpretationStackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="mPageIndividualFeatures">
         <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="1">
           <widget class="QgsSymbolButton" name="mLineStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Fill style</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QgsSymbolButton" name="mFillStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="mCheckRespectLayerSymbology">
            <property name="toolTip">
             <string>If checked, then features will be shown in profile charts using the same symbols or colors as they use on 2D maps.</string>
            </property>
            <property name="text">
             <string>Respect layer's symbology</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Line style</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QgsSymbolButton" name="mMarkerStyleButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Marker style</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mPageContinuousSurface">
         <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="0" colspan="2">
           <widget class="QStackedWidget" name="mSymbologyStackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="mPageLine">
             <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Line style</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsSymbolButton" name="mSurfaceLineStyleButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="mPageFill">
             <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Limit</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsSymbolButton" name="mSurfaceFillStyleButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsDoubleSpinBox" name="mElevationLimitSpinBox">
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="minimum">
                 <double>-99999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>99999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Fill style</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="mCheckBoxShowMarkersAtSampledPoints">
            <property name="text">
             <string>Show markers at sampled points</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mStyleComboBox"/>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QgsSymbolButton" name="mSurfaceMarkerStyleButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Marker style</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mElevationGroupBox</tabstop>
  <tabstop>mComboClamping</tabstop>
  <tabstop>mScaleZSpinBox</tabstop>
  <tabstop>mOffsetZSpinBox</tabstop>
  <tabstop>mOffsetDDBtn</tabstop>
  <tabstop>mBindingGroupBox</tabstop>
  <tabstop>mComboBinding</tabstop>
  <tabstop>mExtrusionGroupBox</tabstop>
  <tabstop>mExtrusionSpinBox</tabstop>
  <tabstop>mExtrusionDDBtn</tabstop>
  <tabstop>mTypeComboBox</tabstop>
  <tabstop>mCheckRespectLayerSymbology</tabstop>
  <tabstop>mMarkerStyleButton</tabstop>
  <tabstop>mLineStyleButton</tabstop>
  <tabstop>mFillStyleButton</tabstop>
  <tabstop>mStyleComboBox</tabstop>
  <tabstop>mSurfaceLineStyleButton</tabstop>
  <tabstop>mSurfaceFillStyleButton</tabstop>
  <tabstop>mCheckBoxShowMarkersAtSampledPoints</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
